### PR TITLE
Fixes bug 1237589 - Removed remnants of automatic-emails code from es…

### PR DESCRIPTION
**Pull request by adngdb**
… module.

### CODE REVIEW

File: socorro/unittest/external/es/test_index_creator.py [hunk 1]
```diff
@@ -22,19 +20,6 @@ def __init__(self, *args, **kwargs):
 
         self.config = self.get_tuned_config(IndexCreator)
 
-    def tearDown(self):
```
Comment (peterbe) [inline socorro/unittest/external/es/test_index_creator.py]: Don't you need to also delete the `socorro_test_temp` directory?

File: socorro/unittest/external/es/test_index_cleaner.py [hunk 2]
```diff
@@ -102,17 +102,16 @@ def test_delete_old_indices(self):
...
 
-        assert self.index_client.exists('socorro_emails')
+        assert self.index_client.exists('socorro_test_temp')
 
         api = IndexCleaner(self.config)
         api.delete_old_indices()
 
         # Verify the email index is still there.
-        ok_(self.index_client.exists('socorro_emails'))
+        ok_(self.index_client.exists('socorro_test_temp'))
```
Comment (peterbe) [inline socorro/unittest/external/es/test_index_cleaner.py]: I see a `create_index()` but no `delete_index()`. Thus, isn't this unit test leaking?

**Comment by peterbe** [comment]
r+ if you address the problem with leaving a temp index around. It's best practice to not have tests create things that it doesn't delete.

**Comment by peterbe** [comment]
I suggest you rebase against master and let the tests run again.

File: socorro/unittest/external/es/test_index_creator.py (hunk 1 above)
Comment (adngdb) [inline socorro/unittest/external/es/test_index_creator.py]: That's done in the base class' ``tearDown`` method.

Comment (adngdb) [inline socorro/unittest/external/es/test_index_creator.py]: Actually, wait, ``socorro_test_temp`` is not created in this file, so why should it be removed here?

File: socorro/unittest/external/es/test_index_cleaner.py (hunk 2 above)
Comment (adngdb) [inline socorro/unittest/external/es/test_index_cleaner.py]: It is done line 43, using ``self.indices`` as the list of indices to delete.

**Pull request merged**